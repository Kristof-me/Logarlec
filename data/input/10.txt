load map.data
create student s1 r1
create transistor t1 s1
create transistor t2 s1
use s1 t1
status t1 pair
status t2 pair
drop s1 t1
status t1 pair
status t2 pair
status t1 inventory
status t2 inventory