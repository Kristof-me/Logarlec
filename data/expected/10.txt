t2
t1
t2
t1
r1
s1
